pkg/apis/app/v1/kieapp_types.go

```diff
@@ -359,17 +366,16 @@ type BuildTemplate struct {
 }
 
 type CommonConfig struct {
-	ApplicationName    string       `json:"applicationName,omitempty"`
-	Auth               AuthTemplate `json:"auth,omitempty"`
-	Version            string       `json:"version,omitempty"`
-	ImageTag           string       `json:"imageTag,omitempty"`
-	Product            string       `json:"product,omitempty"`
-	KeyStorePassword   string       `json:"keyStorePassword,omitempty"`
-	AdminPassword      string       `json:"adminPassword,omitempty"`
-	ControllerPassword string       `json:"controllerPassword,omitempty"`
-	ServerPassword     string       `json:"serverPassword,omitempty"`
-	MavenRepo          string       `json:"mavenRepo,omitempty"`
-	MavenPassword      string       `json:"mavenPassword,omitempty"`
+	ApplicationName    string `json:"applicationName,omitempty"`
+	Version            string `json:"version,omitempty"`
+	ImageTag           string `json:"imageTag,omitempty"`
+	Product            string `json:"product,omitempty"`
+	KeyStorePassword   string `json:"keyStorePassword,omitempty"`
+	AdminPassword      string `json:"adminPassword,omitempty"`
+	ControllerPassword string `json:"controllerPassword,omitempty"`
+	ServerPassword     string `json:"serverPassword,omitempty"`
+	MavenRepo          string `json:"mavenRepo,omitempty"`
```

Comment:
should this be moved to template constants? elsewhere?

Reply:
I think we should leave this credentials configurable

Reply:
agree, what about  `MavenRepo` though?